translations/servers.json

```diff
@@ -0,0 +1,1553 @@
+[
+ {
+  "countryCode": "au",
+  "languages": {
+   "cy": "Awstralia",
+   "dsb": "Awstralska",
+   "hsb": "Awstralska",
+   "pt_BR": "Austrália",
+   "zh_TW": "澳洲",
+   "nl": "Australië",
+   "ja": "オーストラリア",
+   "fr": "Australie",
+   "ru": "Австралия",
+   "oc": "Austràlia",
+   "el": "Αυστραλία",
+   "uk": "Австралія",
+   "sk": "Austrália",
+   "de": "Australien"
+  },
+  "cities": [
+   {
+    "city": "Melbourne",
+    "languages": {
+     "ru": "Мельбурн",
+     "de": "Melbourne City",
+     "en": "City of Melbourne",
+     "fr": "Ville de Melbourne",
+     "it": "Città di Melbourne",
+     "ja": "メルボルン市"
+    }
+   },
+   {
+    "city": "Sydney",
+    "languages": {
+     "ru": "Сидней",
+     "uk": "Сідней",
+     "ja": "シドニー",
+     "el": "Σίδνεϊ",
+     "zh_TW": "雪梨"
+    }
+   }
+  ]
+ },
+ {
+  "countryCode": "at",
+  "languages": {
+   "pt_BR": "Áustria",
+   "sk": "Rakúsko",
+   "zh_TW": "奧地利",
+   "ru": "Австрия",
+   "ja": "オーストリア",
+   "nl": "Oostenrijk",
+   "uk": "Австрія",
+   "fr": "Autriche",
+   "de": "Österreich",
+   "oc": "Àustria",
+   "el": "Αυστρία",
+   "cy": "Awstria",
+   "dsb": "Awstriska",
+   "hsb": "Awstriska"
+  },
+  "cities": [
+   {
+    "city": "Vienna",
+    "languages": {
+     "cy": "Fienna",
+     "dsb": "Wien",
+     "el": "Βιέννη",
+     "hsb": "Wien",
+     "sk": "Viedeň",
+     "sq": "Vjenë",
+     "fr": "Vienne",
+     "nl": "Wenen",
+     "oc": "Viena",
+     "ja": "ウィーン",
+     "ru": "Вена",
+     "uk": "Відень",
+     "de": "Wien",
+     "pt_BR": "Viena",
+     "zh_TW": "維也納"
+    }
+   }
+  ]
+ },
+ {
+  "countryCode": "be",
+  "languages": {
+   "pt_BR": "Bélgica",
+   "ru": "Бельгия",
+   "sk": "Belgicko",
+   "sq": "Belgjika",
+   "uk": "Бельгія",
+   "zh_TW": "比利時",
+   "co": "Belgica",
+   "cy": "Gwlad Belg",
+   "de": "Belgien",
+   "dsb": "Belgiska",
+   "el": "Βέλγιο",
+   "fr": "Belgique",
+   "hsb": "Belgiska",
+   "it": "Belgio",
+   "ja": "ベルギー",
+   "nl": "België",
+   "oc": "Belgica"
+  },
+  "cities": [
+   {
+    "city": "Brussels",
+    "languages": {
+     "cy": "Dinas Brwsel",
+     "en_CA": "City of Brussels",
```

Comment:
Same here, we should fix the "City of" variants

Reply:
This is confusing on wikipedia too. There 2 bruxelles entities:
- https://en.wikipedia.org/wiki/Brussels
- https://en.wikipedia.org/wiki/City_of_Brussels

I'm pointing to the second one. Do you suggest the first one?

Reply:
Yes, the city should be the first one (capital of the municipality).

Reply:
Sorry, I misread. 1st is region, 2nd is city, so we should use the 2nd, and fix manually where necessary.